client/dashboard/sites/welcome-modal/opt-in-welcome-modal.tsx

```diff
@@ -0,0 +1,84 @@
+import { userPreferenceQuery, userPreferenceMutation } from '@automattic/api-queries';
+import { useSuspenseQuery, useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	__experimentalSpacer as Spacer,
+	Button,
+	ExternalLink,
+	Icon,
+	Modal,
+} from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { wordpress } from '@wordpress/icons';
+import { useAnalytics } from '../../app/analytics';
+import ComponentViewTracker from '../../components/component-view-tracker';
+import illustrationUrl from './opt-in-welcome-modal-illustration.png';
+import './opt-in-welcome-modal.scss';
+
+const preferenceName = 'hosting-dashboard-opt-in-welcome-modal-dismissed' as const;
+
+export default function OptInWelcomeModal() {
+	const { recordTracksEvent } = useAnalytics();
+	const isLargeViewport = useViewportMatch( 'small', '>=' );
+	const { data: isDismissedPersisted } = useSuspenseQuery( userPreferenceQuery( preferenceName ) );
+	const { mutate: updateDismissed, isPending: isDismissing } = useMutation(
+		userPreferenceMutation( preferenceName )
+	);
+
+	const isDismissed = isDismissedPersisted || isDismissing;
+
+	const handleDismiss = () => {
+		recordTracksEvent( 'calypso_dashboard_opt_in_welcome_modal_dismiss_click' );
+		updateDismissed( new Date().toISOString() );
+	};
+
+	if ( isDismissed ) {
+		return null;
+	}
+
+	return (
+		<Modal
+			className="dashboard-opt-in-welcome-modal"
+			shouldCloseOnEsc={ false }
+			isDismissible={ false }
+			__experimentalHideHeader
+			onRequestClose={ handleDismiss }
+		>
+			<ComponentViewTracker eventName="calypso_dashboard_opt_in_welcome_modal_impression" />
+			<HStack alignment="stretch" spacing={ 6 }>
+				<VStack alignment="flex-start" spacing={ 4 } style={ { flex: 1 } }>
+					<Icon icon={ wordpress } />
+					<Text className="dashboard-opt-in-welcome-modal__title" as="h1">
+						{ __( 'Meet the new WordPress.com hosting dashboard' ) }
```

Comment:
I thought we'd agreed somewhere to treat this as a proper noun

Source pgz0xU-kw-p2#comment-470

```suggestion
						{ __( 'Meet the new WordPress.com Hosting Dashboard' ) }
```

Reply:
It followed the comment on DOTCOM-15531

Reply:
Yes, it should be Hosting Dashboard.

Reply:
Updated by https://github.com/Automattic/wp-calypso/pull/107799/commits/799c88b2b18b37ba30ed03a6eeab18c7685134e0.